crates/wallet/tests/wallet.rs

```diff
@@ -116,8 +116,11 @@ fn wallet_is_persisted() -> anyhow::Result<()> {
 
     run(
         "store.db",
-        |path| Ok(bdk_file_store::Store::create_new(DB_MAGIC, path)?),
-        |path| Ok(bdk_file_store::Store::open(DB_MAGIC, path)?),
+        |path| Ok(bdk_file_store::Store::create(DB_MAGIC, path)?),
+        |path| {
+            let (_, store) = bdk_file_store::Store::load(DB_MAGIC, path)?;
+            Ok(store)
+        },
```

Comment:
nit: This could be condensed slightly. similar suggestion below in `wallet_load_checks`.

```suggestion
        |path| Ok(bdk_file_store::Store::load(DB_MAGIC, path)?.1),
```

Reply:
I will change it. Consider I have a bias towards explicit semantics.

Reply:
Fixed by b93cec96d60f71025795678dda0c29586f7a27b6